doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc

```diff
@@ -0,0 +1,79 @@
+== Configuring Image Scanning
+
+{product-title} manages vulnerability scanning of container images. When the OpenShift provider is added, OpenShift images from the internal registry are discovered. To enable image scanning, perform the following configuration steps:
+
+. Navigate to menu:Compute[Containers > Providers].
+. Select the *OpenShift* provider checkbox.
```

Comment:
To be a little more clear, I'd suggest writing this as something like "Select an *OpenShift* provider." or "Select the checkboxes of the OpenShift providers to configure image scanning on."